tests/detect-vlan-id/writepcap.py

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env python3
+from scapy.all import *
```

Comment:
Can we also have the scapy version as a comment here in the script? Makes it easier to not lose the information :)